lib/template-mixins.js

```diff
@@ -254,6 +254,7 @@ module.exports = function (options, deprecated) {
                 legend: t(legendValue),
                 legendClassName: legendClassName,
                 role: opts.type === 'radio' ? 'radiogroup' : 'group',
+                ariaRequired: opts.type === 'radio' ? 'true' : '',
```

Comment:
I don't think this needs the ternary operator here as it's only being used as a condition in the templates and not as the actual value.

I'd slim it down to just:

```js
ariaRequired: opts.type === 'radio',
```

since it only needs to be a boolean and not a pseudo-boolean string.